.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Purpose of this PR
+
+* Description of feature/change.
+* Links to screenshots, design docs, user docs, etc.
+
+### Testing status
+
+* Explanation of what’s tested, how tested and existing or new automation tests.
+* Can include manual testing by self.
+* Specify test plans.
+* Rarely acceptable to have no testing.
+
+### Technical risk
+
+* Overall product level assessment of risk of change. 
+* Need technical risk & halo effect.
```

Comment:
I would drop this section, typically PRs are related to issues, which are discussed before.

We're not really using the WIP PR or speculative PR styles.